test/test_helper.rb

```diff
@@ -16,6 +16,17 @@ def using_mysql?
 end
 
 require File.expand_path("../dummy/config/environment.rb",  __FILE__)
+if Gem::Version.new(ActiveRecord::VERSION::STRING) >= Gem::Version.new('5.0.0.beta1')
+  ActiveSupport.on_load(:action_controller) do
+    ActionController::TestCase.send(:include, Rails::Controller::Testing::TestProcess)
+    ActionController::TestCase.send(:include, Rails::Controller::Testing::TemplateAssertions)
+
+    ActionDispatch::IntegrationTest.send(:include, Rails::Controller::Testing::TemplateAssertions)
+    ActionDispatch::IntegrationTest.send(:include, Rails::Controller::Testing::Integration)
+
+    ActionView::TestCase.send(:include, Rails::Controller::Testing::TemplateAssertions)
+  end
```

Comment:
Copied wholesale from [rails-controller-testing](https://github.com/rails/rails-controller-testing/blob/master/lib/rails-controller-testing.rb). I don't know why putting it here works but requiring the library doesn't. I've also tried `require: false` in the Gemfile and manually requiring it instead but that doesn't help. Removing this code, I get:
```
NoMethodError: assigns has been extracted to a gem. To continue using it,
        add `gem 'rails-controller-testing'` to your Gemfile.
```
Not knowing if this is caused by something in PaperTrail or an issue with rails-controller-testing itself (or Rails 5), I'm not sure whether I should raise an issue over there.